src/vs/workbench/browser/parts/editor/multiEditorTabsControl.ts

```diff
@@ -1303,7 +1298,7 @@ export class MultiEditorTabsControl extends EditorTabsControl {
 				break;
```

Comment:
This can be simplified: `isSelected` on group view can allow to pass in the index as `number`

Reply:
Well the editor is needed anyways after that but I can rewrite